CONTRIBUTION.md

```diff
@@ -48,4 +71,26 @@ In the main.tf file location, initialize Terraform by running the following:
 terraform init -reconfigure -upgrade -backend-config="access_key=<STORAGE_BUCKET_ACCESS_KEY>" -backend-config="secret_key=<STORAGE_BUCKET_SECRET_KEY>"
 ```
 
-This will throw an error ("Failed to query available provider packages") which can be ignored since we are using the local provider build.
\ No newline at end of file
+This will throw an error ("Failed to query available provider packages") which can be ignored since we are using the local provider build.
+
+## Code Contributions
+
+To make your contribution, follow these steps:
+
+1. Make sure you are working on the latest version of the `main` branch.
```

Comment:
Is this step 1. relevant if we are going to fork the repo?

Reply:
I think its relevant because if the contributor is working on an older version of the main branch, the contribution he wants to make might have been done already in the latest version. The fork might also be not synced with the latest version from the original repo

Reply:
But then maybe it should be moved to after the "fork the repo" step?